src/components/Delegates/Delegations/DelegationsContainer.tsx

```diff
@@ -59,16 +86,34 @@ function DelegationsContainer({
                 </TableRow>
               </TableHeader>
               <TableBody>
-                {delegators.length === 0 ? (
-                  <div className="w-full p-4">None found</div>
-                ) : (
-                  delegators.map((delegation) => (
-                    <DelegationFromRow
-                      key={delegation.from}
-                      delegation={delegation}
-                    />
-                  ))
-                )}
+                {/* @ts-ignore */}
+                <InfiniteScroll
```

Comment:
@ferrodri do you have any ideas on how to better embed infinite scroll into the table body so that it's not interpreted as a single column
![CleanShot 2024-03-14 at 15 57 26](https://github.com/voteagora/agora-next/assets/20438019/085ac681-dc41-4129-80dd-b043bbdc719f)

Reply:
Solved on this commit :) https://github.com/voteagora/agora-next/pull/169/commits/b74184e140704cb1b7b3ecfaf9680656edb37f1f

Reply:
thank you!